alerting-config-service-api/src/main/proto/org/hypertrace/alerting/config/service/v1/metric_anomaly_event_condition.proto

```diff
@@ -0,0 +1,103 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.alerting.config.service.v1;
+
+message MetricAnomalyEventCondition {
+  MetricSelection metric_selection = 1;
+  repeated ViolationCondition violation_condition = 2;
+  string evaluation_window_duration = 3; // iso duration string
+}
+
+message MetricSelection {
+  Attribute metric_attribute = 1;
+  MetricAggregationFunction metric_aggregation_function = 2;
+  string metric_aggregation_interval = 3; //iso duration string PT15S
+  Filter filter = 4;
+}
+
+message Attribute {
+  string key = 1;
+  string scope = 2;
+}
+
+enum MetricAggregationFunction {
+  METRIC_AGGREGATION_FUNCTION_TYPE_UNSPECIFIED = 0;
+  METRIC_AGGREGATION_FUNCTION_TYPE_SUM = 1;
+  METRIC_AGGREGATION_FUNCTION_TYPE_AVG = 2;
+  METRIC_AGGREGATION_FUNCTION_TYPE_AVGRATE = 3; // avgrate period is assumed to be 1sec
+}
+
+message Filter {
+  oneof filter {
+    LeafFilter leaf_filter = 1;
+    CompositeFilter composite_filter = 2;
+  }
+}
+
+message LeafFilter {
+  LhsExpression lhs_expression = 1;
+  ValueOperator value_operator = 2;
+  RhsExpression rhs_expression = 3;
+}
+
+enum ValueOperator {
+  VALUE_OPERATOR_UNSPECIFIED = 0;
+  VALUE_OPERATOR_EQ = 1;
+}
+
+enum LogicalOperator {
+  LOGICAL_OPERATOR_UNSPECIFIED = 0;
+  LOGICAL_OPERATOR_AND = 1;
+  LOGICAL_OPERATOR_OR = 2;
+}
+
+message CompositeFilter {
+  LogicalOperator logical_operator = 1;
+  repeated Filter child_filters = 2;
+}
+
+
+message LhsExpression {
+  oneof value {
+    Attribute attribute = 1;
+  }
+}
+
+message RhsExpression {
+  oneof value {
+    string string_value = 1;
+  }
+}
+
+enum Severity {
+  SEVERITY_UNSPECIFIED = 0;
+  SEVERITY_CRITICAL = 1;
+  SEVERITY_WARNING = 2;
+}
+
+message ViolationCondition {
+  oneof condition {
+    StaticThresholdCondition static_threshold_condition = 2;
+    BaselineThresholdCondition baseline_threshold_condition = 3;
+  }
+}
+
+message StaticThresholdCondition {
+  StaticThresholdOperator operator = 1;
```

Comment:
Here as well, can we go for `1, 2, and 3` instead of `1, 2, and 4`?

Reply:
yes we can, this file is copied as is from alert-engine